### PR TITLE
feat(fa): Create staff

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...
https://app.asana.com/0/1201301944561348/1201257203547389

## What
Adds ability to create user in Veita if current staff is admin
Adds ui for creating staff

## Why
Needed feature for admin system

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+      <Box marginBottom={2}>
+        <Checkbox
+          name={'admin'}
+          label="Stjórnandi (admin)"
+          checked={state.roles.includes(StaffRole.ADMIN)}
+          hasError={state.hasError && state.roles.length === 0}
+          onChange={(event) => {
+            changeStaffAccess(StaffRole.ADMIN, event.target.checked)
+          }}
+          strong={false}
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx:175]: filled is false but not in the other checkbox, is the other one automatically filled? 💭

File: apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        />
+      </Box>
+      {state.hasError && state.roles.length === 0 && (
+        <Text color="red600" fontWeight="semiBold" variant="small">
+          Það þarf að velja réttindi
+        </Text>
+      )}
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx:182]: maybe have a little animation on the appearance on the error message? 🪄

File: apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx (hunk 1 above)
Comment (Rikkiabb) [reply apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx:175]: Nope it is default on false, will remove from the top one :)

File: apps/financial-aid/web-veita/src/routes/Users/users.tsx [hunk 3]
```diff
@@ -32,15 +39,33 @@ export const Users = () => {
...
       isLoading={loading}
       loader={<ApplicationOverviewSkeleton />}
     >
-      <Box className={`contentUp delay-25`} marginTop={15} key={'Notendur'}>
-        <Text as="h1" variant="h1" marginBottom={[2, 2, 4]}>
+      <Box
+        className={`${styles.header} contentUp delay-25`}
+        marginTop={15}
+        marginBottom={[2, 2, 4]}
+        key={'Notendur'}
```
[the remark this answers is not in the document]
Comment (Rikkiabb) [reply apps/financial-aid/web-veita/src/routes/Users/users.tsx:56]: Removed ;)

**Review by MargretFinnboga — APPROVED** [review]
osom 🐾   👏

**Pull request merged**